### ISSUE TITLE
Check if OKO calibration edge is just global logit temperature

### ISSUE THREAD

**Issue by devWhyqueue** [devWhyqueue/master-thesis#3]
When interpreting benchmark results: OKO dominates on calibration metrics (ECE etc). Before crediting method itself, check if edge just comes from better global logit temperature (i.e. run temperature scaling on baseline logits and compare).
